### ISSUE TITLE
http: provide friendly cookie and header operations

### ISSUE THREAD

**Issue by ssibrahimbas** [ssibrahimbas/istanbul#27]
We should be able to write code like this:

```typescript
const token req.cookie.token; // get cookie
req.cookie.token = token; // set cookie
```

```typescript
const token = req.headers.authorization; // get header
req.headers.authorization = token; // set header
```

**Comment by ssibrahimbas** [comment]
Good job.

Various request header properties such as secure, http only and expire should be supported. Also it looks like you skipped headers.

**Comment by ssibrahimbas** [comment]
fully resolved in 6b153ad0b525230219a03a72b504301486c0a9e3
